### PR TITLE
Refactor constants

### PR DESCRIPTION
**Pull request by kumuwu**

### CODE REVIEW

File: bacon/commons/Constants.swift [hunk 1]
```diff
@@ -8,8 +8,9 @@
 
 import Foundation
 
-class Config {
-    static let defaultIsExpenditure = true
+class Constants {
+    // Add Transaction Page Constants
+    static let defaultMode = TransactionType.expenditure
```
Comment (fterh) [inline bacon/commons/Constants.swift]: Should call it defaultType?

File: bacon/commons/Constants.swift [hunk 2]
```diff
@@ -8,8 +8,9 @@
...
     static let defaultCategoryString = "Food"
```
Comment (fterh) [inline bacon/commons/Constants.swift]: Use raw value

File: bacon/controller/AddTransactionViewController.swift [hunk 3]
```diff
@@ -10,27 +10,27 @@ import UIKit
 
 class AddTransactionViewController: UIViewController {
 
-    var isExpenditure = Config.defaultIsExpenditure
-    private var selectedCategory = Config.defaultCategory
+    var mode = Constants.defaultMode
```
Comment (fterh) [inline bacon/controller/AddTransactionViewController.swift]: Should rename from mode to type

File: bacon/controller/AddTransactionViewController.swift [hunk 4]
```diff
@@ -10,27 +10,27 @@ import UIKit
...
+    private var selectedCategory = Constants.defaultCategory
```
Comment (fterh) [inline bacon/controller/AddTransactionViewController.swift:14]: var or let?

**Review by GilgameshTC — APPROVED** [review]

**Pull request merged**